analyses/cell-type-ewings/template_notebooks/01-marker-genes.Rmd

```diff
@@ -33,6 +33,10 @@ Cells with a z-score for any marker gene > 0 are classified as tumor cells.
 ```{r}
 project_root <- here::here()
 renv::load(project_root)
+
+knitr::opts_chunk$set(
+  dev = "jpeg"
+)
```

Comment:
Just curious how much space savings this gave, if you know?

Reply:
I don't remember... but I don't think it was a huge difference.